extensions/amp-story-subscription/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'ampproject/wg-stories'}],
```

Comment:
Plz specify OWNERS manually, there are more people than should be OWNERS in this group.

I think you, @mszylkowski , and `{name: 'newmuis', notify: true}`, `{name: 'gmajoulet', notify: true},` would be cool

Reply:
Done. Also added Caleb.